ncat segfault when making connection do unix domain socket

Downstream report: https://bugzilla.redhat.com/show_bug.cgi?id=1957178

rhel-9-0
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testHttpsInstanceDoS (__main__.TestConnection)
Traceback (most recent call last):
  File "/usr/lib64/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/work/bots/make-checkout-workdir/test/verify/check-connection", line 313, in testHttpsInstanceDoS
    self.assertGreater(active, 45)
  File "/usr/lib64/python3.9/unittest/case.py", line 1229, in assertGreater
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib64/python3.9/unittest/case.py", line 670, in fail
    raise self.failureException(msg)
AssertionError: 1 not greater than 45
Journal extracted to TestConnection-testHttpsInstanceDoS-rhel-9-0-127.0.0.2-2401-FAIL.log.gz
Traceback (most recent call last):
  File "/work/bots/make-checkout-workdir/test/verify/check-connection", line 313, in testHttpsInstanceDoS
    self.assertGreater(active, 45)
AssertionError: 1 not greater than 45

# Result testHttpsInstanceDoS (__main__.TestConnection) failed
# 1 TEST FAILED [23s on 1-ci-srv-03]
not ok 19 test/verify/check-connection TestConnection.testHttpsInstanceDoS
```
First occurrence: 2021-05-05T11:37:45.473462 | revision 6bd369b3590c0dd5e66ab8c5c60b981275e2f7c8
Times recorded: 1
Latest occurrences:

- 2021-05-05T11:37:45.473462 | revision 6bd369b3590c0dd5e66ab8c5c60b981275e2f7c8

rhel-9-0
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testUnitLifecycle (__main__.TestConnection)
bash: line 1:  3593 Segmentation fault      (core dumped) nc -U /run/cockpit/wsinstance/http.sock 2>&1
Traceback (most recent call last):
  File "/usr/lib64/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/work/bots/make-checkout-workdir/test/verify/check-connection", line 288, in testUnitLifecycle
    self.assertIn("Permission denied", out)
  File "/usr/lib64/python3.9/unittest/case.py", line 1098, in assertIn
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib64/python3.9/unittest/case.py", line 670, in fail
    raise self.failureException(msg)
AssertionError: 'Permission denied' not found in ''
Journal extracted to TestConnection-testUnitLifecycle-rhel-9-0-127.0.0.2-2501-FAIL.log.gz
Core dumps downloaded to /work/bots/make-checkout-workdir/TestConnection-testUnitLifecycle-rhel-9-0-127.0.0.2-2501-FAIL.core
Traceback (most recent call last):
  File "/work/bots/make-checkout-workdir/test/verify/check-connection", line 288, in testUnitLifecycle
    self.assertIn("Permission denied", out)
AssertionError: 'Permission denied' not found in ''

# Result testUnitLifecycle (__main__.TestConnection) failed
# 1 TEST FAILED [29s on 1-ci-srv-03]
not ok 24 test/verify/check-connection TestConnection.testUnitLifecycle
```
First occurrence: 2021-05-05T11:38:24.828932 | revision 6bd369b3590c0dd5e66ab8c5c60b981275e2f7c8
Times recorded: 1
Latest occurrences:

- 2021-05-05T11:38:24.828932 | revision 6bd369b3590c0dd5e66ab8c5c60b981275e2f7c8

--- a/naughty/rhel-9/1993-ncat-segfault
+++ b/naughty/rhel-9/1993-ncat-segfault
@@ -1,2 +1,0 @@
-# testUnitLifecycle (__main__.TestConnection)
-bash: line 1:  * Segmentation fault      (core dumped) nc -U /run/cockpit/wsinstance/http.sock 2>&1

--- a/naughty/rhel-9/1993-ncat-segfault-2
+++ b/naughty/rhel-9/1993-ncat-segfault-2
@@ -1,4 +1,0 @@
-# testHttpsInstanceDoS (__main__.TestConnection)
-Traceback (most recent call last):
-*
-AssertionError: 1 not greater than 45

